pal03377/Athena-New
# Rollback Pydantic version from V2 to V1

### Motivation and Context


The automatic update to Pydantic V2 caused some trouble in the new apted module

### Description

Fixed pedantic version to 1.10.13

### Steps for Testing

1. Open Athena
2.  Delete poetry.lock file
3. execute `poetry install`
4. Check if module is working
5. Do the same for the other modules (where I did not change anything) to verify that they also work

### Screenshots
